notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb

```diff
@@ -232,7 +231,7 @@
         "         \"geoId/065300\",   # Oakland\n",
         "         \"geoId/0644000\"]  # Los Angeles"
       ],
-      "execution_count": null,
+      "execution_count": 3,
```

Comment:
Should the execution counts not be null? I guess it depends on whether we want the outputs to be visible by default or not?